tdrs-backend/Pipfile

```diff
@@ -21,10 +21,11 @@ pytest-mock = "*"
 pytest-factoryboy = "*"
 
 [packages]
+setuptools = "<58.0"
```

Comment:
@abottoms-coder what is this needed for?

Reply:
The latest version of setuptools (60) gets rid of a function used by several of our dependencies including ipython.